grid2op/tests/test_alert_trust_score.py

```diff
@@ -0,0 +1,1767 @@
+# Copyright (c) 2019-2023, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
```

Comment:
I don't understand what is done here. Is this all copy paste from the AlertReward ? 

If so what's the point, as everything is tested in AlertReward already ? What is the difference between the _AlertTrustScore and the AlertReward.

These are the only things that needs to be tested. Making copy paste of previous tests is not testing more stuff.

Reply:
I stop my reading here

Reply:
This is to test _AlertTrustScore.

I don't understand what is misleading you here.

What is tested is the cumulated reward, the min and max possible cumulated reward, the score. This was the format of test chosen before hand

Reply:
The point was to test that all cases covered by  tests on the AlertReward are covered in the _AlertTrustScore. 
The attributes that are tested must be different though